tendrl/ceph_integration/manager/__init__.py

```diff
@@ -39,7 +37,13 @@ def main():
     complete = gevent.event.Event()
 
     def shutdown():
-        LOG.info("Signal handler: stopping")
+        try:
+            payload = {"message": "Signal handler: stopping"}
+            event.Event(Message(Message.priorities.INFO,
+                                Message.publishers.CEPH_INTEGRATION,
+                                payload))
+        except event.EventFailed:
+            print(traceback.format_exc())
```

Comment:
@r0h4n This will give error when socket failed, if socket failed we cant log anything, so we cant log it